logging/console/src/layer.ts

```diff
@@ -0,0 +1,44 @@
+import type { EventPayload, Layer } from '@rango-dev/logging-types';
+
+import { levelToConsoleStyle, levelToName } from './helpers';
+
+function formatMessage(payload: EventPayload) {
+  return [
+    `%c[%s] %O %O`,
+    levelToConsoleStyle(payload.level),
+    levelToName(payload.level),
+    payload.message,
+    payload.data,
+  ];
+}
+
+export function layer(): Layer {
+  return {
+    handler(payload) {
+      const message = formatMessage(payload);
+
+      switch (payload.level) {
+        case 0:
```

Comment:
It is better to explicitly use the level name instead of the number

Reply:
Thanks. I changed it to use Level enum.